managed/models/database.go

```diff
@@ -838,6 +838,50 @@ var databaseSchema = [][]string{
 		FROM services
         WHERE service_type = 'mongodb';`,
 	},
```

Comment:
I think we need to prefix all onboarding related tables with `onboarding_`

Reply:
okay